chemie/rentalservice/templates/rentalservice/new_object.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'chemie/base.html' %}
```

Comment:
Try to avoid doing unrelated changes in the same PR, i guess this should be a commit in the rental service branch instead of here =)

Reply:
This was not the intention. My bad!